engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/ProcessValidatorSpec.scala

```diff
@@ -180,6 +180,43 @@ class ProcessValidatorSpec extends FunSuite with Matchers with Inside {
 
   }
 
+  test("Valid dynamic property access when available") {
+    val baseDefinitionCopy = baseDefinition.copy(
+      expressionConfig = baseDefinition.expressionConfig.copy(
+        disableDynamicPropertyAccess = false))
+
+    val correctProcess = EspProcessBuilder
+      .id("process1")
+      .exceptionHandler()
+      .source("id1", "source")
+      .filter("filter1", "#input.plainValue == 1")
+      .filter("filter2", "#input['plainValue'] == 1")
+      .sink("id2", "#input", "sink")
+
+    val compilationResult = validate(correctProcess, baseDefinitionCopy)
+
+    compilationResult.result should matchPattern {
+      case Valid(_) =>
+    }
+  }
+
+
+  test("Valid dynamic property access when disabled") {
+    val correctProcess = EspProcessBuilder
+      .id("process1")
+      .exceptionHandler()
+      .source("id1", "source")
+      .filter("filter1", "#input.plainValue == 1")
```

Comment:
What about this filter? Do we want to disable access here or not?

Reply:
Well, it's not necessary. Already changed that. As first I thought, it will be better to show that problem is with 2nd filter, but you're right.